databases/elasticsearch/Portfile

```diff
@@ -139,7 +139,7 @@ curl -XDELETE \"http://localhost:9200/my_first_index\"
 
 Each Elasticsearch shard is a Lucene index; aim for 10-50 GB per shard.
 
-${name} is tested with the JDK provided in port openjdk${java.version}. Add these
+${name} is tested with the JDK provided in port openjdk${java.fallback}. Add these
```

Comment:
```suggestion
${name} is tested with the JDK provided in port ${java.fallback}. Add these
```

Reply:
Thanks for catching that, fixed in a7b356ed8